src/frontend/components/VideoPlayer/index.tsx

```diff
@@ -114,31 +148,14 @@ const VideoPlayer = ({
         poster={thumbnailUrls[resolutions[0]]}
         tabIndex={-1}
       >
-        {resolutions.map((size) => (
+        {resolutions.map((size, index) => (
           <source
-            key={video.urls!.mp4[size]}
+            key={`url_${index}`}
             size={`${size}`}
             src={video.urls!.mp4[size]}
             type="video/mp4"
           />
         ))}
-
-        {timedTextTracks
-          .filter((track) => track.is_ready_to_show)
-          .filter((track) =>
-            [timedTextMode.CLOSED_CAPTIONING, timedTextMode.SUBTITLE].includes(
-              track.mode,
-            ),
-          )
-          .map((track) => (
-            <track
-              key={track.id}
-              src={track.url}
-              srcLang={track.language}
-              kind={trackTextKind[track.mode]}
-              label={languages[track.language] || track.language}
-            />
-          ))}
```

Comment:
I'm still not a fan of this "fix" that does not look like it.
Yes the problem will no longer occure here but we are not fixing the intrinsec issue causing an error to be throw If a component mounts two map and some  key changes.

Reply:
We finally found the problem originates from how video.js works. Indeed, regardless of if you inject track with the API or with React, videojs redraws it owns DOM. When using react and updating track, we hence have a difference between the videojs DOM and the react virtual DOM, causing rendering crashes.

Reply:
What if you destroy the videojs player before creating a new one (IE : adding appropriate dependancies to the useEffect) ?